code/modules/admin/secrets.dm

```diff
@@ -684,13 +684,16 @@ GLOBAL_DATUM_INIT(admin_secrets, /datum/admin_secrets, new)
 				everyone!")
 			log_admin("[key_name(usr)] has Un-Fully Immersed everyone.")
 		if("animalsentience")
+			message_admins("[key_name_admin(usr)] has made all mobs sentient")
+			log_admin("[key_name(usr)] has made all mobs sentient.")
 			for(var/mob/living/simple_animal/L in GLOB.alive_mob_list)
 				var/turf/T = get_turf(L)
 				if(!T || !is_station_level(T.z))
 					continue
-				if((L in GLOB.player_list) || L.mind || (L.flags_1 & HOLOGRAM_1))
+				if(L.mind || (L.flags_1 & HOLOGRAM_1))
```

Comment:
this isn't log related, you should add it to changelog

Reply:
Espee doesn't changelog shit, following their example I'm not going to so I can see what I can sneak past maintainers

Reply:
add it to the changelog. It isn't just used for code review, it helps us get a general idea of what might have messed up if stuff breaks as a result